Python/cpuinfo.c

```diff
@@ -0,0 +1,110 @@
+/*
+ * Naive CPU SIMD features detection.
+ *
+ * See Modules/black2module.c.
+ */
+
+#include "Python.h"
+#include "pycore_cpuinfo.h"
+
+#include <stdbool.h>
```

Comment:
It seems that `stdbool` has been defined in `Python.h` Maybe this can be removed it.

Reply:
I prefer not to include the entire Python if I don't need to. And I prefer explicit includes.